map_view/display-drawing-status/src/main/java/com/esri/samples/display_drawing_status/DisplayDrawingStatusSample.java

```diff
@@ -87,18 +87,14 @@ public void start(Stage stage) {
       Envelope envelope = new Envelope(bottomLeftPoint, topRightPoint);
       mapView.setViewpoint(new Viewpoint(envelope));
 
-      mapView.addDrawStatusChangedListener(e -> {
-        // check to see if draw status is in progress
-        if (e.getDrawStatus() == DrawStatus.IN_PROGRESS) {
-          // reset progress bar as in progress
-          progressBar.setProgress(-100.0);
-
-          // check to see if draw status is complete
-        } else if (e.getDrawStatus() == DrawStatus.COMPLETED) {
-          // set progress bar as complete
-          progressBar.setProgress(100.0);
-        }
-      });
+      // bind progress bar status and map view draw status
+      progressBar.progressProperty().bind(
```

Comment:
I wouldn't say I was a particular fan of this binding - to my mind this is a very artificial binding as drawing is either in progress or complete with no in between so a progress bar values don't make sense to me. 

A progress indicator makes more sense to me - it shows something is happening. With the following you can show one when drawing is in progress and hide it when drawing is done.
```
var progressIndicator = new ProgressIndicator();
progressIndicator.visibleProperty().bind(mapView.drawStatusProperty().isEqualTo(DrawStatus.IN_PROGRESS));
```
I think this is also a more understandable binding.

![progress](https://user-images.githubusercontent.com/5682200/188885668-0e167e2c-d261-4884-a0f0-e01d826d29c5.gif)

If I was doing the linear progress bar I would probably just use an invalidation listener e.g.
```
    mapView.drawStatusProperty().addListener(o -> {
      if (mapView.getDrawStatus() == DrawStatus.IN_PROGRESS) {
        progressIndicator.setProgress(-1);
      } else {
        progressIndicator.setProgress(1);
      }
    });
```
to me this is more readable code.

@Rachael-E maybe you can comment on the sample design with respect to progress bar vs progress indicator.

Reply:
I agree @colinanderson , the progress bar here doesn't add anything to the sample UI here, a progress indicator would be more appropriate (implemented similarly to other samples: placed centrally, overlying the map view (default behaviour). Good time to make that change!